cmd/statusd/whispercfg.go

```diff
@@ -57,13 +51,6 @@ func whisperConfig(nodeConfig *params.NodeConfig) (*params.NodeConfig, error) {
 		}
 	}
 
-	// RPC configuration
-	// TODO(adam): clarify all these IPC/RPC/HTTPHost
-	if !*httpEnabled {
-		nodeConfig.HTTPHost = "" // HTTP RPC is disabled
-	}
-	nodeConfig.HTTPHost = "0.0.0.0"
```

Comment:
This may be important when exposing HTTP JSON-RPC interface when running in Docker. If that's the case, we should add `-httphost` flag to make it explicit.

Reply:
are we using http json rpc only for testing? if so it makes sense to change default value

Reply:
I believe so but I am not sure about status-react config options when using status-go as a lib.